roles/openshift_images_common/tasks/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Pulling common docker images
+  shell: docker pull {{ osi_docker_registry }}/openshift3_beta/{{ item }}:{{ osi_image_version }}
```

Comment:
If using the rhel defaults, this shouldn't be necessary.  Also, we'd most likely want to use openshift_registry_url for these by default (or openshift.node.registry_url in my configTemplates branch).